TodoList.Web/Startup.cs

```diff
@@ -34,6 +34,7 @@ public void ConfigureServices(IServiceCollection services)
             services.ConfigureSecurity();
             services.ConfigureIdentity();
             services.ConfigureStorage(Configuration);
+            services.ConfigureSocialAuthentication(Configuration);
```

Comment:
Please fix the formatting.

Reply:
You mean change it to `ConfigureAuthentication`?